programs/cleos/main.cpp

```diff
@@ -2523,14 +2630,26 @@ int main( int argc, char** argv ) {
    auto trxSubcommand = push->add_subcommand("transaction", localized("Push an arbitrary JSON transaction"));
    trxSubcommand->add_option("transaction", trx_to_push, localized("The JSON string or filename defining the transaction to push"))->required();
 
+   trxSubcommand->add_flag("-d,--dont-broadcast", tx_dont_broadcast, localized("don't broadcast transaction to the network (just print to stdout, essentially only validates)"));
```

Comment:
can we dedup this with `add_standard_transaction_options` so that these two options are only managed/inserted in one place in the code?